--- a/common.mk
+++ b/common.mk
@@ -410,13 +410,13 @@
 
   ## TLDR: Leave this variable as "opt" while the remaining are set to "copt". Otherwise,
   ## the correct flags may not be picked up when using gcc on a ARM64 OSX machine
-  opt := -march=native
-  COMPILE_OPT_SUPPORTED := $(shell $(CC) $(opt) -dM -E - < /dev/null 2>&1 1>/dev/null)
-  ifndef COMPILE_OPT_SUPPORTED
-    CFLAGS += $(opt)
-  else
-    CFLAGS := $(filter-out $(opt), $(CFLAGS))
-  endif
+  # opt := -march=native
+  # COMPILE_OPT_SUPPORTED := $(shell $(CC) $(opt) -dM -E - < /dev/null 2>&1 1>/dev/null)
+  # ifndef COMPILE_OPT_SUPPORTED
+  #   CFLAGS += $(opt)
+  # else
+  #   CFLAGS := $(filter-out $(opt), $(CFLAGS))
+  # endif
 
   copt := -mcpu=apple-m1
   COMPILE_OPT_SUPPORTED := $(shell $(CC) $(copt) -dM -E - < /dev/null 2>&1 1>/dev/null)
