--- a/rules.mk
+++ b/rules.mk
@@ -57,6 +57,12 @@
 $(LIBNAME)_impl_double.o: $(UTILS_DIR)/weight_functions_double.h
 $(LIBNAME)_impl_float.o: $(UTILS_DIR)/weight_functions_float.h

+gridlink_%_double.o: gridlink_%_double.c gridlink_%_double.h
+	$(CC) -DDOUBLE_PREC $(GRIDLINK_CFLAGS) $(INCLUDE) $(EXTRA_INCL) -c $< -o $@
+
+gridlink_%_float.o: gridlink_%_float.c gridlink_%_float.h
+	$(CC) -DNDOUBLE_PREC $(GRIDLINK_CFLAGS) $(INCLUDE) $(EXTRA_INCL) -c $< -o $@
+
 %_double.o: %_double.c %_double.h
 	$(CC) -DDOUBLE_PREC $(CFLAGS) $(INCLUDE) $(EXTRA_INCL) -c $< -o $@
